app/models/employee.rb

```diff
@@ -20,5 +30,24 @@ def update_phone(value)
     self.save
   end
 
+  protected
+  def set_naming_format
```

Comment:
1) We dont need to explicitly check the value for nil `self.middle_name!=nil`, we just use `if self.middle_name`  
2) We dont need to call self `self.first_name`, simply `first_name` will suffice  
3) rather than explicitly defining the value `first_name=first_name.capitalize` we use the bang version to change the value `first_name.capitalize!` 
4)we use a single line syntax whenever possible like in this case `middle_name.capitalize! if middle_name`